### PR TITLE
add lang config for Bulgarian, Finnish, Ukrainian, Indonesian, Slovak, Turkish and Maori

### LINKED ISSUE #3330: Open a Bulgarian language track on Weblate
### Description

Hey, I am enjoying Overseer a huge deal!
My use-case is strictly tied to language (letting older relatives make requests). I am willing to help translate and will be happy to spend some time on that, but I can't see a track for **Bulgarian** in Weblate.use case

### Desired Behavior

The Weblate page to have **Bulgarian** as a possible translation language for Overseer

### Additional Context

_No response_

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

### LINKED ISSUE #3658: Maori language
### Description

I'm happy to do the translation

### Desired Behavior

All front end strings in Māori

### Additional Context

_No response_

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

### LINKED ISSUE #3723: Turkish Display Language Support
### Description

I would like to contribute to the translation of the UI to Turkish. Currently Turkish is not listed on the Weblate page.

### Desired Behavior

Display Language being able to set to Turkish

### Additional Context

_No response_

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

### LINKED ISSUE #3765: Add Slovenian language to Weblate
### Description

Please add Slovenian language to Webplate, so we can help translating.

### Desired Behavior

Please add Slovenian language to Webplate, so we can help translating.

### Additional Context

_No response_

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

### PR DESCRIPTION
**Pull request by jordanjones98**
#### Description

Add config for Bulgarian, Finnish, Ukrainian, Indonesian, Turkish, Slovak and Maori

Indonesian, Turkish, Slovak and Maori need to be added to weblate.

NOTE: this PR will not build until #3597 is merged. This PR references files that are created in that PR. Once that PR is merged into develop this one can have develop merged into it, be tested locally, and it will pass.

#### To-Dos

- [ ] Successful build `yarn build`
- [ ] Translation keys `yarn i18n:extract`
- [ ] Database migration (if required)

#### Issues Fixed or Closed

- Fixes #3330 
- Fixes #3723 
- Fixes #3723 
- Fixes #3658 
- Fixes #3765

### CODE REVIEW

**Comment by avidenic** [comment]
Just a a small detail - #3765 is about Slovenian language (language code `sl`) but this PR is about Slovak (language code `sk`) so it should probably not be linked here.

**Comment by danshilm** [comment]
Thank you for the PR again!
The weblate PR has been merged, so I was looking into merging this but I see that you added files for the languages that don't exist on weblate yet. Can we remove those languages to allow weblate itself to create the files in the PR it will eventually make when the languages are added there to be translated? It would be pretty weird for a user to select a language only for them to see nothing translated in that language.

That would leave the languages with the codes 'bg' (Bulgarian), 'fi' (Finnish), 'he' (Hebrew), 'hi' (Hindi), 'ro' (Romanian) and 'uk' (Ukrainian) to be added.

**Comment by jordanjones98** [comment]
@avidenic good catch, thank you for that, per @danshilm I am probably just going to remove the extra languages here for now. I will get this done tonight.

**Comment by jordanjones98** [comment]
@danshilm stuff came up, this is later than I wanted, but those langs are removed!

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -72,6 +72,8 @@ const loadLocaleData = (locale: AvailableLocale): Promise<any> => {
       return import('../i18n/locale/sr.json');
     case 'sv':
       return import('../i18n/locale/sv.json');
+    case 'uk':
+      return import('../i18n/locale/uk.json');
```
Comment (danshilm) [inline src/pages/_app.tsx:76]: Not sure why you removed Bulgarian and Finnish from the list?
And I think when removing the languages, you didn't notice that Hebrew (hi), Hindi (hi), Romanian (ro) are already available from Weblate after #3597 had been merged. Please add those 3 back.

Comment (jordanjones98) [reply src/pages/_app.tsx:86]: I have been busy, but I got all of this sorted out.

**Review by sct — APPROVED** [review]
Thanks! LGTM!

**Pull request merged**